README.md

```diff
@@ -29,6 +29,7 @@ Update version in Chart.yaml and run `helm dependency update`
 
 ```bash
 helm template charts/common -f example/values-minimal.yaml
+helm template charts/common -f example/values-minimal-secret.yaml
```

Comment:
This pattern will quickly hurt (postgres, secrets, sngle container, multi container, custom specs, multi svc,ing etc etc.
I think the absolute minimal for deploy/cron is enough for this repo IMHO.

Reply:
I agree. The file was useful when testing the new functionality, but it does not need to be added to the codebase.